src/components/SignUp/index.tsx

```diff
@@ -292,6 +283,8 @@ const SignUpFormComponent: React.FC<SignUpFormProps> = ({
                         <div
                             className={cr('cr-sign-up-form__group', {
                                 'cr-sign-up-form__group--focused': nicknameFocused,
+                                'cr-sign-up-form__group--errored': nickname.length !== 0 &&
```

Comment:
```suggestion
                                'cr-sign-up-form__group--errored': nickname.length &&
```
this is enough to check for length